### PR TITLE
install pipelines without scrapping the classpath script

### PR DESCRIPTION
**Pull request by estute**
This pr installs the pipelines plugin (workflow-aggregator), which requires the workflow-cps-global-lib plugin. This dependency does the same thing https://github.com/edx/jenkins-configuration/blob/master/src/main/groovy/1addJarsToClasspath.groovy- namely injecting the ivy jar into the current classpath, so that subsequent groovy incantations run under the jenkins process can make use of the grape dependency system. Since plugins are loaded prior to groovy init system, Jenkins complained about two conflicting attempts to overload the class loader.

Basically, this change will do the following:

* If the workflow-cps-global-lib plugin is installed, skip the class loader manipulation and carry on to the next script. You can still use grapes, because the plugin is managing that (see: https://github.com/jenkinsci/workflow-cps-global-lib-plugin/blob/master/src/main/java/org/jenkinsci/plugins/workflow/cps/global/GrapeHack.java#L45)
* For instances when this plugin is not installed, use the 1addJarsToClasspath.groovy

### CODE REVIEW

**Comment by estute** [comment]
CC @pwnage101

File: src/main/groovy/1addJarsToClasspath.groovy [hunk 1]
```diff
@@ -13,46 +13,59 @@ import jenkins.model.*
...
+// which duplicates this script, but since the plugin is loaded before the groovy
+// scripts are initialized, will cause conflicts.
+boolean cpsLibInstalled = Jenkins.instance.getPluginManager().getPlugins().any { plugin ->
+    plugin.toString() == 'Plugin:workflow-cps-global-lib'
 }
+if (cpsLibInstalled) {
+    logger.info('The workflow-cps-global-lib plugin is installed.')
+    logger.info('No need to add ivy into the classpath again')
+    logger.info('Carrying on to the next init script')
+} else {
```
Comment (michaelyoungstrom) [inline src/main/groovy/1addJarsToClasspath.groovy]: Hmm... I know that we currently only use this script for adding ivy to the classpath, but since it supports adding multiple jars, it feels weird to escape the whole script when this plugin exists. Perhaps this logic should instead go towards L53 when the filename is ivy? Also... is there any other way of checking if ivy has already been added?
